CONTRIBUTING.md

```diff
@@ -43,6 +43,104 @@ To contribute your work you need to...
 4. Push to the Branch (`git push origin newBranchName`)
 5. Open a Pull Request
 
+### Commit Message Format
```

Comment:
Imho this should be moved to the Sphinx guideline (GitHub pages) and only link from here to the guideline to avoid building up a "second source".

CONTRIBUTION should only summarize what is needed to contribute.